### PR TITLE
Dustin

### PR DESCRIPTION
**Pull request by yourcrazyex**
I explained all this in HMO discussion chat

### CODE REVIEW

File: common/deities/01_canaanite.txt [hunk 1]
```diff
@@ -173,14 +177,14 @@ deity_yam = {
 		}
 	}
 	icon = deity_cr_car_yam
-	passive_modifier = { monthly_military_experience_modifier = deity_monthly_military_experience_modifier_svalue }
+	passive_modifier = { ship_repair_at_sea = deity_ship_repair_at_sea }
```
Comment (x3igh7) [inline common/deities/01_canaanite.txt:180]: no repair at sea, please revert

Comment (yourcrazyex) [reply common/deities/01_canaanite.txt:180]: I didn't touch this file. How can that happen? 

I changed it, I'll try to push it again

Comment (x3igh7) [reply common/deities/01_canaanite.txt]: It might be because you haven't pull the latest code or because we weren't pointing to the right branch

**Comment by x3igh7** [comment]
If you wouldn't mind summarizing the changes here that would be helpful - mostly so I can copy + paste the highlights into the change log later

File: common/deities/01_canaanite.txt (hunk 1 above)
Comment (x3igh7) [reply common/deities/01_canaanite.txt]: Okay I think this is all fixed now

**Comment by x3igh7** [comment]
Brought back slave revolts so slave happiness matters (thanks Stephan). You will be warned of a revolt if 25 or more slaves have 15% or less happiness in a single tile. The revolt can be avoided if you take action.
· Slowed down base pop assimilation rates (thanks Stephan). Slowed the assimilation rate for the monarchy laws. Slowed assimilation and conversion rates for theatres and temples and for governor policies. Assimilation can now be done via a mix of theatres, cultural decisions, governor policies, inventions and laws.
· Slightly nerfed the effects of cultural decisions and greatly reduced their cost. The PI cost is now no longer static but proportional to the size of the culture's population. These decisions will be important for the happiness of your various cultural populations.
· Reduced cost of Indian spiritual path decisions.
· Added a slave happiness cultural decision for integrated cultures.
· Made some changes to religion. There are now three laws that allow you to focus on many gods or just one. State Pantheon consolidates and codifies your nation's religious traditions. State Syncretism promotes exchanges and mixing between divine traditions, regardless of their origin. Henotheism commands the exclusive adoration of one, jealous god, who shall come before all others. If you choose syncretism, I had set up an easy mechanism to add foreign deities to your pantheon (Expand Pantheon), but did not implement it because I did not understand the code at the time. I think I do now, but my doctor has told me never to mod anything ever again so I can't exactly disobey, now can I??? Maybe some brave modder can take up the slack.
· New law called Customs Tariffs that boosts trade routes.
· Reduced cost of gift slave, gift relic, invite hunting, send letter, tribal duel, task character, etc.
· Removed 'ship repair at sea' effect from some sneaky Canaanite deity.
· Buffed some shit inventions.

**Review by x3igh7 — APPROVED** [review]

**Pull request merged**